View/Elements/sql_log_panel.ctp

```diff
@@ -41,6 +41,19 @@ if (isset($debugKitInHistoryMode)) {
 					echo ' ' . __d('debug_kit', 'No query logs.');
 				endif;
 			else:
+				$hashes = array();
+				$duplicate = 0;
+				foreach ($queryLog['queries'] as $k => $v) {
+					$hash = sha1($v['query']);
+					if (!isset($hashes[$hash]) || $hashes[$hash] !== $v['affected']) {
+						$hashes[$hash] = $v['affected'];
+						continue;
+					}
+					$duplicate++;
+
+					$queryLog['queries'][$k]['query'] = '<span style="color: red"">' . $v['query'] . '</span>';
```

Comment:
I agree. Was just gonna wait for more feedback before giving it a finalize touch.